<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurveyEvaluationDialogBase</class>
 <widget class="QDialog" name="SurveyEvaluationDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Survey evaluation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QComboBox" name="mStratumIdComboBox"/>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="mVavailComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="mDistLabel">
       <property name="text">
        <string>Dist</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="mWidthLabel">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mSampleLayerLabel">
       <property name="text">
        <string>Sample layer</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="mDistComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mCatchLabel">
       <property name="text">
        <string>Catch</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mSampleLayerComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mStratumIdLabel">
       <property name="text">
        <string>Stratum id</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="mVavailLabel">
       <property name="text">
        <string>Vertical availability</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="mWidthComboBox"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="mCatchComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mArealAvailabliltyLabel">
       <property name="text">
        <string>Areal availability</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="mArealAvailabilityComboBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="mBaselineLayerClipComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mBaselineClipLayerLabel">
       <property name="text">
        <string>Baseline clip layer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mSpeciesVulnerabilityLabel">
       <property name="text">
        <string>Species vulnerability</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="mSpeciesVulnerabilitySpinBox">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SurveyEvaluationDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SurveyEvaluationDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
